Fix misplaced #ifdef. A closing brace is always required
for switch() and an empty default is a bug.

--- a/main.c	2004-11-21 19:00:54.000000000 +0100
+++ b/main.c	2015-10-25 23:08:33.060565151 +0100
@@ -173,12 +173,12 @@
             if (ev.xexpose.count == 0);
             handle_expose_event(&ev.xexpose);
             break;
-        default:
 #ifdef SHAPE
+        default:
             if (have_shape && ev.type == shape_event)
                 handle_shape_event((XShapeEvent *) & ev);
-        }
 #endif
+        }
 
     }
     return 1;
